src/decoder/stateless/h265.rs

```diff
@@ -1159,19 +1159,21 @@ where
                 let sps = self.codec.parser.parse_sps(&nalu)?;
                 self.codec.max_pic_order_cnt_lsb = 1 << (sps.log2_max_pic_order_cnt_lsb_minus4 + 4);
 
+                let mut pending_pps = Vec::new();
+                std::mem::swap(&mut self.codec.pending_pps, &mut pending_pps);
```

Comment:
`let pending_pps = std::mem::take(&mut self.codec.pending_pps);` should do the same and lets you make `pending_pps` unmutable.

Reply:
Thank you!